src/components/ProgressBar/styles.tsx

```diff
@@ -0,0 +1,70 @@
+import COLORS from '@/styles/colors';
+import { H4 } from '@/styles/text';
+import styled from 'styled-components';
+
+export const ProgressBarContainer = styled.div`
```

Comment:
Can we change the border weight of the circles and the body to be a bit lighter? It feels slightly heavier than it is on the Figma

Reply:
Border weight and line (I'm assuming that's what you mean by the body) is the same as it is on Figma, 5px. I do think it looks off, but we should discuss that, because many auth screens also seem bigger than they should be. This may be a screen size issue, or a Figma issue.